src/enforce/runners/run/index.js

```diff
@@ -1,7 +1,22 @@
 // @flow
+/**
+ * A function which returns whether a combination of
+ * rule + value is true or false
+ *
+ * @param {any} value - the value being tested
+ * @param {string} key the name of the rule being run
+ * @param {string} tests an object containing the group of tests in the current run
```

Comment:
tests and rules aren't strings.
To define types in jsdoc, see [typedef](http://usejsdoc.org/tags-typedef.html)

Reply:
Whoops. Duplicated the line and forgot to edit the types. Good catch